mevolib/fetch/BioSeqs.py

```diff
@@ -72,10 +76,8 @@ class BioSeqs:
     source.
 
     Attributes:
-        data  (dict)
-            Dictionary where the SeqRecord objects are stored.
-        _report  (Private[list])
-            List of tuples with the information of the source or sources of the SeqRecord objects stored at 
+        data  (dict): Dictionary where the SeqRecord objects are stored.
+        _report  (Private[list]): List of tuples with the information of the source or sources of the SeqRecord objects stored at 
```

Comment:
Arguments and variables that start by `_` are considered as private (e.g.: `_report`). I think this issue is still open (remove private attributes from description).

https://github.com/JAlvarezJarreta/MEvoLib/pull/1/commits/81968f9f749f563add4a6991f096251b5dff420b#r979440740